utils/validate-facility-ids/index.ts

```diff
@@ -0,0 +1,37 @@
+import { difference } from 'lodash';
+import { SqlDbDataSource } from '@ukef/dtfs2-common/sql-db-connection';
+import { mongoDbClient } from './database-client';
+import { getUniqueFeeRecordFacilityIds, getUniqueTfmFacilitiesUkefFacilityIds } from './helpers';
+
+const run = async (): Promise<void> => {
+  const dataSource = await SqlDbDataSource.initialize();
+  try {
+    const tfmFacilitiesUkefFacilityIds = await getUniqueTfmFacilitiesUkefFacilityIds();
+    const feeRecordFacilityIds = await getUniqueFeeRecordFacilityIds(dataSource);
+
+    const tfmFacilitiesFacilityIdsNotInFeeRecord = difference(tfmFacilitiesUkefFacilityIds, feeRecordFacilityIds);
```

Comment:
Nit pick but I think this should end with an 's'

Reply:
This is now removed but I didn't include an 's' at the end because `FeeRecord` is a table